suite-native/module-accounts/src/components/AccountDetailGraphHeader.tsx

```diff
@@ -41,13 +46,21 @@ const hasPriceIncreasedAtom = atom(get => {
     return percentageChange >= 0;
 });
 
-const CryptoBalance = ({ accountSymbol }: { accountSymbol: NetworkSymbol }) => {
+const CryptoBalance = ({ accountKey }: { accountKey: AccountKey }) => {
+    const account = useSelector((state: AccountsRootState) =>
```

Comment:
We need this because the account header balance was calculated from the value of Graph `selectedPointAtom`. For testnet accounts, it was displaying zero or balance of the last displayed graph (very confusing). 

I've refactored it to hide completely AccountDetailGraphHeader for testnet accounts to keep performance and make the code clearer. 274ce39a2